presidio-analyzer/README.MD

```diff
@@ -16,7 +16,7 @@ Named Entity Recognition and other types of logic to detect PII in unstructured
 
 Use the following button to deploy presidio analyzer to your Azure subscription.
 
-[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fmicrosoft%2Fpresidio%2FV2%2Fpresidio-analyzer%2Fdeploytoazure.json)
+[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fmicrosoft%2Fpresidio%2Fmain%2Fpresidio-analyzer%2Fdeploytoazure.json)
 
 TODO: change this link to main branch once merged (#2765).
```

Comment:
```suggestion
```

Reply:
the deploy to azure button links to an ARM template. we need to update the branch to main

Reply:
It's already working after the branching, checked it